python/triton/compiler/code_generator.py

```diff
@@ -1253,7 +1262,8 @@ def visit_Call(self, node):
 
         if fn in self.builtin_namespace.values():
             args = map(_unwrap_if_constexpr, args)
-        return fn(*args, **kws)
+        ret = fn(*args, **kws)
+        return _apply_to_tuple_values(ret, lambda x: x) if _is_namedtuple(type(ret)) else ret
```

Comment:
@peterbell10 Oh, I meant it's being converted here.

Reply:
I see, so the `_is_namedtuple` path should never actually be hit from `visit_Return` only from here. Makes sense.